.travis.yml

```diff
@@ -1,40 +1,19 @@
-sudo: required
-dist: trusty
-language: generic
-env:
-  - ROS_DISTRO=indigo
-  - ROS_DISTRO=jade
-  - ROS_DISTRO=kinetic
-notifications:
-  email:
-    on_success: change # [always|never|change], default=change
-    on_failure: change # [always|never|change], default=always
+sudo: true
 
-before_install:
-  - CI_SOURCE_PATH=$(pwd)
-  - REPOSITORY_NAME=${PWD##*/}
-  - echo "Testing branch $TRAVIS_BRANCH of $REPOSITORY_NAME"
-
-install:
-  - source <(wget -O- https://raw.githubusercontent.com/tue-robotics/tue-env/master/installer/scripts/bootstrap-ros-$ROS_DISTRO)
-  - tue-get install ros-$REPOSITORY_NAME
-  - source ~/.tue/setup.bash # source all target setup files
+language: cpp
 
-before_script:
-  # remove the tue-get version of our repo with the travis one
-  - cd "$TUE_ENV_DIR/repos/https_/github.com/tue-robotics"
-  - mv "$REPOSITORY_NAME.git" "$REPOSITORY_NAME.old"
-  - ln -s "$CI_SOURCE_PATH" "$REPOSITORY_NAME.git"
+services:
+  - docker
 
-  # install travis' package dependencies
-  - cd "$TUE_SYSTEM_DIR/src"
-  - rm *
-  - tue-get update ros-$REPOSITORY_NAME
-  - ls -l
-
-  # go to the catkin workspace
-  - cd "$TUE_SYSTEM_DIR"
+before_install:
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
```

Comment:
Can we make this build trigger as well when tue-env is updated? 
Via http://eng.rightscale.com/2015/04/27/dependent-builds-in-travis.html or https://github.com/mernst/plume-lib/blob/master/bin/trigger-travis.sh